_includes/dochead.html

```diff
@@ -34,7 +34,7 @@
     var trackOutboundLink=function(n){ga("send","event","outbound","click",n,{hitCallback:function(){document.location=n}})};
   </script>
  </head>
- <body>
+ <body class="article">
```

Comment:
Its not clear why you need to override at this level? Is this because all of the CSS in the API .html files are causing display conflicts? Im wondering whats blocking you from setting these changes at the sect1 class level?